tests/test_slurm_system.py

```diff
@@ -8,8 +8,14 @@
 @pytest.fixture
 def slurm_system():
     nodes = [
-        SlurmNode(name="nodeA001", partition="main", state=SlurmNodeState.UNKNOWN_STATE),
-        SlurmNode(name="nodeB001", partition="main", state=SlurmNodeState.UNKNOWN_STATE),
+        SlurmNode(name="node-115", partition="main", state=SlurmNodeState.UNKNOWN_STATE),
```

Comment:
I see you made an assumption about node names. You assume that node names follow the format node-[num]. However, this is not always guaranteed. We need to cover cases with node[num] as well. I am curious if the updated code covers this case too.

Reply:
the code I put is just the older version of the function. What's important is that it works for IL1 (while your code doesnt)
Once again, you can approve the MR and then change the function as you please but check that it still works for IL1

Reply:
@jeffnvidia can't you just add this other case here as well? does it add any more complexity?

Reply:
I covered both cases now